paasta_tools/paasta_remote_run.py

```diff
@@ -147,39 +232,55 @@ def remote_run_start(args):
         sys.stderr = open('/dev/null', 'w')
 
     paasta_print('Scheduling a task on Mesos')
-    scheduler = AdhocScheduler(
-        service_name=service,
-        instance_name=instance,
-        instance_type=instance_type,
-        cluster=cluster,
-        system_paasta_config=system_paasta_config,
-        soa_dir=soa_dir,
-        reconcile_backoff=0,
-        dry_run=args.dry_run,
-        staging_timeout=args.staging_timeout,
-        service_config_overrides=overrides_dict,
+
+    mesos_address = '{}:{}'.format(
+        mesos_tools.get_mesos_leader(), mesos_tools.MESOS_MASTER_PORT
     )
-    driver = create_driver(
+
+    paasta_config = system_paasta_config.get_paasta_native_config()
+    # TODO: implement DryRunExecutor?
+    mesos_executor = MesosExecutor(
+        role='*',
```

Comment:
eventually we will have that value in paasta native config or a separate paasta taskproc config, current value is for testing.